java-base/Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN microdnf update && \
     microdnf install \
     # Obviously needed to run Java programs
     java-${PRODUCT}-openjdk-headless \
+    # Includes tools such as jps or jmap (which is e.g. needed by the Spark UI, see https://github.com/stackabletech/spark-k8s-operator/discussions/340)
+    # We consider them usefull enough to justify putting them in all Java images
```

Comment:
```suggestion
    # We consider them useful enough to justify putting them in all Java images
```

Reply:
Where is the spellcheck action when we need it the most? :D

Reply:
:)